### DISCUSSION TITLE
Data Structures Preview is out 🤟

### DISCUSSION THREAD

**Discussion by y9vad9** [y9vad9/kotlin-course#80] [Announcements]
Hola, everyone 👋
I've just added a few updates to the course, let's sum it up:
  • Added [preview of data structure block](https://course.y9vad9.com/docs/block-2/intro) (linked [PR](https://github.com/y9vad9/kotlin-course/pull/58))
  • Added blog and first article about [Extension Oriented Design](https://course.y9vad9.com/en/blog/extension-oriented-design) in [English](https://course.y9vad9.com/en/blog/extension-oriented-design) and [Ukrainian](https://course.y9vad9.com/blog/extension-oriented-design) (linked [PR](https://github.com/y9vad9/kotlin-course/pull/64))
  • Links are improved (there is no more block-1/intro, but kotlin/basics/intro; linked [PR](https://github.com/y9vad9/kotlin-course/pull/64))
  • Some swearing in Ukrainian added for error messages 👀

I will be delighted to see your feedback about new updates (to make the course better), so you're welcome.

![image](https://user-images.githubusercontent.com/32961194/203158085-4362422d-4b55-465b-9ad6-19cf2ef434cb.png)
![image](https://user-images.githubusercontent.com/32961194/203158160-401a9554-9cea-48ff-90dd-78b4ec989c68.png)
<img width="1424" alt="image" src="https://user-images.githubusercontent.com/32961194/203158280-6e8215e8-52c3-4eaa-97d6-14c13c9c9651.png">

**Comment by LilBroice** [comment]
Pretty good course with interesting serving but personaly I would add some explanation to the code text. Thats not hard but it may help to remember terminology.

**Comment by y9vad9** [comment]
I've added also topic about [Strings and Chars](https://course.y9vad9.com/docs/kotlin/data-structures/strings) #86
